deluge/tests/test_ui_console_add.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Deluge and is licensed under GNU General Public License 3.0, or later, with
+# the additional special exception to link portions of this program with the OpenSSL library.
+# See LICENSE for more details.
+#
+
+from __future__ import unicode_literals
+
+import argparse
+
+from deluge.tests.test_ui_entry import ConsoleUIBaseTestCase
+from deluge.ui.console.cmdline.commands.add import Command
+
+
+class UICommonTestCase(ConsoleUIBaseTestCase):
```

Comment:
should inherit from `basetest.BaseTestCase` as well for the tests to be run.

Reply:
Done.